_data/tagging-status.yml

```diff
@@ -245,9 +245,13 @@
 
  - name: delarray
    type: package
-   status: unknown
+   status: compatible
+   supported-through: [phase-III,math,table]
+   comments: "Requires the math module.
+     Use of math tagging requires support in external tools."
```

Comment:
As this is not supposed to be a permanent state I suggested we change that line everywhere to

Use of math tagging currently requires ....

And I don't think we need "Requires the math module", this is a leftover from the times we didn't have "supported-through:" Again I think this can be dropped everywhere

@u-fischer @davidcarlisle ?

Reply:
@FrankMittelbach  certainly we don't need to say Requires  the math module,  now we have `supported-through`  adding `currently` OK as well. Alternatively we could remove it all together and just add a global note about `math` support in the paragraph at the top that starts `To use packages or classes together with ...`

Probably simplest to merge this PR as-is, and then we can do the rewording as a global edit over all the package entries separately after the merge?

Reply:
A global note is simpler (one place) but on the other hand the info is useful if you just consult the ist for a single package, so for now I would leave it  in. And yes, merge and then global edit afterwards